index.html

```diff
@@ -1658,6 +1658,26 @@ <h1>Speakers</h1>
                               </div>
                         </div>
 
+                          <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="M Padmal" src="img/Tameesh.jpg">
```

Comment:
Write the same name in alt=" " and use use image: 300x300. Please reference the issue you are solving, if no issue raised then create an issue and reference that issue in this PR.